File: app/src/main/java/com/rafaelmfer/nasaexperience/model/MarsModel.java
package com.rafaelmfer.nasaexperience.model;

public class MarsModel {

    private String sunNumber, sunDateObservation, sunTemperature, sunWindSpeed, sunPressure;

    public MarsModel() {
    }

    public MarsModel(String sunNumber, String sunDateObservation, String sunTemperature, String sunWindSpeed, String sunPressure) {
        this.sunNumber = sunNumber;
        this.sunDateObservation = sunDateObservation;
        this.sunTemperature = sunTemperature;
        this.sunWindSpeed = sunWindSpeed;
        this.sunPressure = sunPressure;
    }

    public String getSunNumber() {
        return sunNumber;
    }

    public void setSunNumber(String sunNumber) {
        this.sunNumber = sunNumber;
    }

    public String getSunDateObservation() {
        return sunDateObservation;
    }

    public void setSunDateObservation(String sunDateObservation) {
        this.sunDateObservation = sunDateObservation;
    }

    public String getSunTemperature() {
        return sunTemperature;
    }

    public void setSunTemperature(String sunTemperature) {
        this.sunTemperature = sunTemperature;
    }

    public String getSunWindSpeed() {
        return sunWindSpeed;
    }

    public void setSunWindSpeed(String sunWindSpeed) {
        this.sunWindSpeed = sunWindSpeed;
    }

    public String getSunPressure() {
        return sunPressure;
    }

    public void setSunPressure(String sunPressure) {
        this.sunPressure = sunPressure;
    }
}

File: app/src/main/java/com/rafaelmfer/nasaexperience/model/MarsModel.kt
package com.rafaelmfer.nasaexperience.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "mars_model")
data class MarsModel(

        @PrimaryKey(autoGenerate = true)
        var id: Int,

        @ColumnInfo(name = "sun_number")
        val sunNumber: String,

        @ColumnInfo(name = "sun_date_observation")
        val sunDateObservation: String,

        @ColumnInfo(name = "sun_temperature")
        val sunTemperature: String,

        @ColumnInfo(name = "sun_wind_speed")
        val sunWindSpeed: String,

        @ColumnInfo(name = "sun_pressure")
        val sunPressure: String
)